upcloud/resource_upcloud_server_test.go

```diff
@@ -13,6 +13,59 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
+func TestUpcloudServer_custom(t *testing.T) {
```

Comment:
Would name it `customPlan` or something like that, as users can also have custom server images :)

Reply:
Good point! I will rename it.